src/angular/styles/_typography.scss

```diff
@@ -645,7 +652,7 @@
     }
 
     &:where(.sbb-lean.sbb-dark) {
-      --sbb-color-weak-accent: var(--sbb-color-milk);
+      --sbb-color-weak-accent: var(--sbb-color-smoke);
```

Comment:
`--sbb-color-weak-accent` is a globally used color that should stay `--sbb-color-milk` in Dark Mode. So please undo this change.

Instead, I suggest adding a variable `--sbb-panel-border-color` in `_typography.scss` and setting it to `var(--sbb-color-weak-accent)` globally.

In the Dark mode, you should override it with `--sbb-panel-border-color: var(--sbb-color-smoke)`.

Let me know if you need help.